validation-test/stdlib/FixedPointDiagnostics.swift.gyb

```diff
@@ -29,53 +29,53 @@ func expectSameType<T>(_: T.Type, _: T.Type) {}
 
 func test_truncatingBitPatternAPIIsStableAcrossPlatforms() {
   // Audit and update this test when adding new integer types.
-  expectSameType(Int64.self, IntMax.self)
-  expectSameType(UInt64.self, UIntMax.self)
+  expectSameType(Int64.self, IntMax.self) // expected-error {{'IntMax' has been renamed to 'Int64'}}  
+  expectSameType(UInt64.self, UIntMax.self) // expected-error {{'UIntMax' has been renamed to 'UInt64'}}
```

Comment:
Hmmm, we should fix this error message to something that tells people what to do instead of using `[U]IntMax`. That's separate from this PR, though.